File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    let answer = 0;
    let vTemp = [];
    for (let i = 0; i <= t.length - p.length; i++) {
        let temp = "";
        for (let j = 0; j < p.length; j++) {
            temp += t[i + j];
        }
        vTemp.push(temp);
    }
    for (let i = 0; i < vTemp.length; i++) {
        if (vTemp[i] <= p) {
            answer++;
            console.log(vTemp[i]);
        }
    }
    return answer;
}

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.cpp
#include <string>
#include<iostream>
#include <vector>

using namespace std;

int solution(string t, string p) {
    int answer = 0;
    vector<string> vTemp;
    for(int i=0;i<t.size()-p.size()+1;i++){
        string temp;
        for(int j=0;j<p.size();j++)
        {
            temp+=t[i+j];   
        }
        vTemp.push_back(temp);
    }
    for(int i=0;i<vTemp.size();i++){
        if(vTemp[i]<=p) { 
            answer++;
        }
        cout<<vTemp[i]<<endl;
    }
 
    return answer;
}